schemas/dol_tables.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS data.dol_ui;
+
+/* Create and define metadata tables */
+CREATE TABLE data.dol_ui (
+    vintage DATE,
+    dt DATE,
+    fips INTEGER references meta.us_fips(fips),
+    variable_name TEXT,
+    value REAL
+
+);
+
+CREATE UNIQUE INDEX dol_ui_key ON data.dol_ui (vintage, dt, fips, variable_name);
```

Comment:
Lets's specify this as `PRIMARY KEY (vintage, dt, fips, variable_name` in the create table so we get a primary key and a unique index

Reply:
Done